docs/develop/api-reference/auth-tokens.md

```diff
@@ -0,0 +1,174 @@
+
+## Permission Objects
+
+These objects define the specific role with cache or topic information and then assigned to a [TokenScope](#tokenscope). 
+
+### CachePermission
+A component of a [TokenScope](#tokenscope) object that defines permissions for a cache.
+
+| Name            | Type                  | Description                                                                           |
+| --------------- |-----------------------|---------------------------------------------------------------------------------------|
+| role           | ReadOnly \| ReadWrite | The type of access granted by the permission.                                         |
+| cache          | `AllCaches` \| String | A permission can be restricted to a select cache by name or AllCaches built-in value. |
+
+For role, using `CacheRole.ReadOnly` allows access to all read data plane APIs on caches (e.g. `get`, `DictionaryGetField`, etc.) defined in the CacheSelector. Using `CacheRole.ReadWrite` allows access for all read and write data plane APIs on the caches defined in the CacheSelector. Custom roles are not supported.
+
+For cache, the value can be the built-in `AllCaches` or a string value containing the name of the cache this permission is for.
+
+### TopicPermission
+A component of a [TokenScope](#tokenscope) object that defines permissions for a token.
+
+| Name            | Type                              | Description                                                                          |
+| --------------- |-----------------------------------|--------------------------------------------------------------------------------------|
+| role           | SubscribeOnly \| PublishSubscribe | The type of access granted by the permission.               |
+| cache          | `AllCache` \| String              | A permission can be restricted to a select cache by name or AllCaches built-in value.              |
+| topic          | `AllCache` \| String              | A permission can be restricted to a select topic by name or AllTopics built-in value. |
+
+For role, there are two managed roles to assign, `TopicRole.PublishSubscribe` and `TopicRole.SubscribeOnly`. Custom roles are not supported. Using the SubscribeOnly role allows only subscribing to topics, whereas using the PublishSubscribe role allows publishing and subscribing to topics.
+
+For cache, only topics within that cache's namespace are allowed by the permission. This can be set to the built-in `AllCache` value or a string specifically naming a cache.
+
+For topic, this can be set to the built-in `AllTopics` value, which gives access to all topics in the cache(s) defined in cache or it can be a string with a specific topic name.
+
+#### TopicScope examples
+This is an example of creating a TokenScope with just CachePermissions.
+
+```javascript
```

Comment:
We shouldn't have code directly embedded here - I think you already pointed out it is temporary but just reiterating so we don't forget.

Reply:
docs code snippets PR: https://github.com/momentohq/client-sdk-javascript/pull/623

Reply:
Yep. There is a plan to get this using the SDK repo examples, but we need to kick this out the door.